### PR TITLE
Update AIB residual mix and total supplier mix EFs names

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [X] Update

**What category or categories does it affect?**

_Electricity_

**What?**

_Add "market-based" and "location-based" to activity_name_

**Why?**

_Make the EU electricity EFs ready for the electricity end point_

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -484,447 +484,447 @@ a87a8413-f75b-46b4-931b-8dc697920aee,Energy,Electricity,electricity-energy_sourc
...
 b89463a6-f9e9-4a64-bc13-f29aaecb238a,Energy,Electricity,electricity-energy_source_grid_mix_production_mix,Production mix,kWh,not-supplied,0.154,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,electricity_generation,AIB,,2017,2016,2016,SK,p,Climatiq,2022/12/02,Emission intensity of production mix as reported. Published by the Association of Issuing Bodies (AIB). Retrieved from 2016 European Residual Mix Report. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.aib-net.org/facts/european-residual-mix/2016
-7ebdf5f2-8616-4d8d-b084-bb8b7a62d30e,Energy,Electricity,electricity-energy_source_residual_mix,Residual mix,kWh,not-supplied,0.0,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,electricity_generation,AIB,,2022,2021,2021,AT,p,Climatiq,2022/10/01,Emission intensity of residual mix as reported. Published by the Association of Issuing Bodies (AIB). Retrieved from 2021 European Residual Mix Report. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.aib-net.org/facts/european-residual-mix/2021
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:487]: 927-487 = 441 changes made.
An odd number! Are these changes complete? There are different numbers of changes for different countries:
11 changes for AT
14 changes for FR
15 changes for BE

**Review by TimCroker — COMMENTED** [review]
927-487 = 441 changes made in total.
An odd number! Are these changes complete? There are different numbers of changes for different countries:
11 changes for AT
14 changes for FR
15 changes for BE

The changes themselves (e.g. add market- to residual) look OK.

**Comment by maryarbab** [comment]
> 927-487 = 441 changes made in total. An odd number! Are these changes complete? There are different numbers of changes for different countries: 11 changes for AT 14 changes for FR 15 changes for BE
> 
> The changes themselves (e.g. add market- to residual) look OK.

@TimCroker yes, the number of changes are different per country, as not all of them were reported in each year.

**Review by AnastasiaLobanova — APPROVED** [review]
Looks good @maryarbab - approved!

**Pull request merged**